src/bin/pgcopydb/dump_restore.c

```diff
@@ -574,10 +545,11 @@ copydb_target_finalize_schema(CopyDataSpec *specs)
 		return false;
 	}
 
+	specs->restoreOptions.section = POST_DATA;
```

Comment:
I still wonder what's the advantage of that approach compared to adding a section argument to the function call to `pg_restore_db`?

Reply:
My point of view is that it is in the end one of the restore options and thus should be part of this structure.

Reply:
That's a good point, sure. The way we edit the RestoreOption just before the call, and only there, still makes me wonder about it. But yeah, fine.